<launch>
	<arg name="pkg_name" value="choose_direction_duckie"/>
	<arg name="node_name" default="choose_direction_node"/>
	<arg name="veh" doc="duckie"/>
	<arg name="local" default="false"/>
    <arg name="config" default="baseline" doc="Specify a config."/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />

    <group ns="$(arg veh)">
        <node if="$(arg local)" name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" clear_params="true" required="true">
        </node>

        <!-- run remote -->
        <node unless="$(arg local)" machine="$(arg veh)" name="$(arg node_name)" pkg="$(arg pkg_name)" type="choose_direction_node.py" output="screen">
        </node>
    </group>
</launch>
